tekton/ci/jobs/tekton-github-tasks-completed.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: github-tasks-completed
+  namespace: tektonci
+  description: |
+    Verifies that a PR has all the tasks completed
+spec:
+  params:
+    - name: body
+      description: The body of the Pull Request
+  stepTemplate:
+    env:
+      - name: HOME
+        value: /tekton/home
+  steps:
+    - name: install-pyyaml
+      image: python:3-alpine
+      script: |
+        pip install pyyaml --user
+    - name: check-github-tasks-completed
+      image: python:3-alpine
+      script: |
+        #!/usr/bin/env python
+        import sys
+        prBodyText = """$(params.body)"""
```

Comment:
I'm wondering if this could be tricked into executing arbitrary python code because Tekton is doing direct string interpolation. Not 100% sure but if my PR body is something like this:

```
foo""" + print("~~evil script~~") + """bar
```

I think it might render to:

```
prBodyText = """foo""" + print("~~evil script~~") + """bar"""
```

I think we can potentially work around this by placing `params.body` into an environment variable first on the container and then accessing the env var from the python code:

```
env:
- name: PULL_REQUEST_BODY
  value: $(params.body)
# ...
    script: |
      # ...
      prBodyText = os.getenv('PULL_REQUEST_BODY')
      # ...
```

Not totally sure this is a risk vector but figured I'd mention it. wdyt?

Reply:
makes sense, the other ci jobs also do the same thing so will make an issue to update them to use env instead of params directly -- thank you!

Reply:
created an issue for this - https://github.com/tektoncd/plumbing/issues/971